src/test/java/com/iexec/sms/secret/app/ApplicationRuntimeSecretControllerTest.java

```diff
@@ -0,0 +1,199 @@
+package com.iexec.sms.secret.app;
+
+import com.iexec.sms.authorization.AuthorizationService;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.mockito.Mockito.*;
+
+class ApplicationRuntimeSecretControllerTest {
```

Comment:
Might add a test with mixed-case checksum of app address since this is a common issue

Reply:
That looks more like an IT than a UT, but you're right.